configure.ac

```diff
@@ -91,7 +91,7 @@ SHLIB_VERSION_ARG=""
 # Finished checking, handle options.
 
 AC_ARG_ENABLE(experimental,
-	AC_HELP_STRING([--enable-experimental], [enable experimental code]))
+	AS_HELP_STRING([--enable-experimental], [enable experimental code]))
 
```

Comment:
What's the difference between `AC_HELP_STRING` and `AS_HELP_STRING`? Why is the latter better than the former?